CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 # Contributing
 
-Thank you for contributing to `zq`! Feel free to
-[post an issue](https://github.com/brimsec/zq/issues),
-fork the repo, or send us a pull request.
+Thank you for contributing to `zq`!
+
+Per [common practice](https://www.thinkful.com/learn/github-pull-request-tutorial/Feel-Free-to-Ask#Feel-Free-to-Ask), please [open an issue](https://github.com/brimsec/zq/issues) before sending a pull request. If you think your ideas might benefit from some refinement via Q&A, come talk to us on [Slack](https://join.slack.com/t/brimsec/shared_invite/zt-cy34xoxg-hZiTKUT~1KdGjlaBIuUUdg) as well.
```

Comment:
In keeping with the rest of the file, use two spaces between sentences and break lines to around 100 columns.
```suggestion
Per [common practice](https://www.thinkful.com/learn/github-pull-request-tutorial/Feel-Free-to-Ask#Feel-Free-to-Ask),
please [open an issue](https://github.com/brimsec/zq/issues) before sending a pull request.  If you
think your ideas might benefit from some refinement via Q&A, come talk to us on
[Slack](https://join.slack.com/t/brimsec/shared_invite/zt-cy34xoxg-hZiTKUT~1KdGjlaBIuUUdg) as well.
```